base/stats.go

```diff
@@ -329,6 +329,8 @@ type DbReplicatorStats struct {
 	PushRejectedCount        *SgwIntStat `json:"sgr_push_rejected_count"`
 	PushDeltaSentCount       *SgwIntStat `json:"sgr_deltas_sent"`
 	DocsCheckedSent          *SgwIntStat `json:"sgr_docs_checked_sent" `
+	NumConnectAttemptsPull   *SgwIntStat `json:"num_connects_pull"`
+	NumReconnectsAbortedPull *SgwIntStat `json:"num_reconnects_aborted_pull"`
```

Comment:
Sorry didn't catch this before, but now I'm seeing them next to other stats, should prefix these with `sgr_` and probably keep the expvar name and prometheus name consistent (i.e: `sgr_num_connect_attempts_pull`)

Reply:
Done